lib/components/home/artwork_rails/artwork_rail.js

```diff
@@ -116,14 +112,39 @@ class ArtworkRail extends React.Component {
     )
   }
 
+  renderViewAllButton() {
+    if (this.expandable() && !this.state.expanded) { return }
+
+    if (this.hasAdditionalContent()) {
+        return (
+          <TouchableHighlight style={styles.viewAllButton} onPress={this.handleViewAll} underlayColor={'gray'}>
+            <Text style={styles.viewAllText}>VIEW ALL</Text>
+          </TouchableHighlight>
+        )
+      }
+
+      // otherwise, use a spacer view
+      return <View style={{ height: 30 }} />
+    }
```

Comment:
The indentation does seem right above, which made it slightly confusing to read.